tool/template-sidebar_nav.html

```diff
@@ -1,107 +1,110 @@
+{% if use_page is undefined %}
+  {% set use_page = currentpage %}
```

Comment:
The `use_page` thing provides a way to show the sidebar from a perspective other than the current page's position in the hierarchy.